atlasdb-commons/src/main/java/com/palantir/util/AssertUtils.java

```diff
@@ -26,7 +26,18 @@
 
 public class AssertUtils {
 
-    private static final Logger log = LoggerFactory.getLogger(AssertUtils.class);
+    /**
+     * Previously, failed asserts logged to this log.
+     * However, this led to the problem that the logger would always be com.palantir.util.AssertUtils,
+     * which is extremely annoying to try to filter based on your logging properties.
+     * (Should it go into the server error log, or maybe the upgrade log, or the import log file?
+     * Can't tell, cause it's all AssertUtils!)
+     *
+     * Until we get all downstream projects off of using defaultLog however,
+     * this will stay, just deprecated.
+     */
+    @Deprecated
+    private static final Logger defaultLog = LoggerFactory.getLogger(AssertUtils.class);
```

Comment:
wiki "Asserts, Validates and Your Code". I'm not arguing whether or not this is a good idea, just that given that there's a bunch of code that uses it, we should make it more useful.